src/analytics/src/main/scala/osmesa/analytics/Footprints.scala

```diff
@@ -495,18 +473,77 @@ object Footprints extends Logging {
                   RasterTileWithKey(k, z, x, y, GTRaster.tupToRaster(newTile, targetExtent))
               }
           }
+        // TODO this is better but doesn't actually work
```

Comment:
@echeipesh, @moradology can you shed some light on what I'm doing wrong here? The `reduceGroups` approach drops tiles, so something isn't being merged properly.

Reply:
It looks like the reason that `newTile.merge` is failing has something to do with merging *into* a `MutableSparseIntTile`...

Reply:
'Twas nodata handling.